src/services/symbolication.rs

```diff
@@ -1704,6 +1727,53 @@ impl SymbolicationActor {
                         });
                     }
 
+                    // If we successfully timed the new stackwalking method earlier, compute its
+                    // stacktraces as well
```

Comment:
I think it makes sense to move the post-processing into it's own function which takes the processing state and whatever else it needs.  That way we are sure that both post-processing are identical and not the source of any diffs.  Plus it makes the function not blow up so much (it also may alleviate the need for the somewhat weird `Option<(ProcessState, Duration)>` thing).

Also I think currently the module list is not included in the comparison, I would make that part of the new function and include it.  Yes this will mean some more cloning of stuff for these cases, but I think that's justifyable.